src/java_bytecode/java_entry_point.cpp

```diff
@@ -167,14 +182,38 @@ bool java_static_lifetime_init(
        it->second.type.id()==ID_code &&
        it->second.mode==ID_java)
     {
-      code_function_callt function_call;
-      function_call.lhs()=nil_exprt();
-      function_call.function()=it->second.symbol_expr();
-      function_call.add_source_location()=source_location;
-      code_block.add(function_call);
+      const irep_idt symbol_name=
+        it->second.symbol_expr().get_identifier();
+      const std::string &symbol_str=id2string(symbol_name);
+      const std::string suffix(".<clinit>:()V");
```

Comment:
this is a candidate to have in a `#define` or `const std::string` somewhere and also adapt other occurrences